data/src/main/java/com/microsoft/azure/kusto/data/ClientImpl.java

```diff
@@ -45,6 +45,8 @@ public class ClientImpl implements Client, StreamingClient {
     private final CloseableHttpClient httpClient;
     private boolean endpointValidated = false;
 
+    private ObjectMapper objectMapper = Utils.getObjectMapper();
```

Comment:
@tanmaya-panda1 ,  though this a good idea to have it initialized once there are discussions if this causes deadlocking (Note that ObjectMapper is a threadsafe class , so technically should not be a problem).  There is no action needed , but something that you can watchout for (https://stackoverflow.com/questions/3907929/should-i-declare-jacksons-objectmapper-as-a-static-field)